cves/kernel/CVE-2021-29266.yml

```diff
@@ -448,7 +449,8 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: This bug was due to a coding mistake and a lack of unit tests. The fix to this issue was
```

Comment:
You could probably make this last section a bit more interesting. Maybe something about how even the smallest of bugs can cause big issues? That said, it is a pretty small bug so it's understandable you don't have too much